pkg/apis/options/loginurlopts/login_url_parameters.go

```diff
@@ -1,4 +1,4 @@
-package options
+package loginurlopts
```

Comment:
Why has this moved?

Reply:
> otherwise get import cycles,
> 
> ```
> package github.com/oauth2-proxy/oauth2-proxy/v7
>         imports github.com/oauth2-proxy/oauth2-proxy/v7/pkg/apis/options
>         imports github.com/oauth2-proxy/oauth2-proxy/v7/tenant
>         imports github.com/oauth2-proxy/oauth2-proxy/v7/tenant/configloader
>         imports github.com/oauth2-proxy/oauth2-proxy/v7/pkg/apis/options: import cycle not allowed
> ```
> 
> basically ConfigLoader (a TenantLoader implementation) config depends on the provider config, which is part of `pkg/apis/options` package, and the options.Options depend on the TenantLoader config which depends upon the ConfigLoader config.
> 
> This is resolved by moving the provider options to a new package.

same thing as above, since provider options depends on loginurlopts,
we can keep the provider opts and loginurlopts in the same package though,

Reply:
resolved, moved the file/s back to its original location.